scripts/variantstore/wdl/extract/test_create_genes_bqloadjson_from_annotations.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import gzip
+import tempfile
+
+from create_genes_bqloadjson_from_annotations import make_annotation_json
+
+dir='variant_annotation_table_test_files/'
+normal_annotated_genes_json=dir + "test_annotated_genes.json.gz"
+expected_output_annotated_genes_bq_json=dir + "expected_test_output_annotated_genes_bq.json.gz"
+
+# An old style (before we separated positions and genes jsons into separate files) - expected to fail
+old_annotated_genes_json=dir + "test_old_annotated_genes.json.gz"
```

Comment:
could bring this into the test method as it's not referenced anywhere else

Reply:
Yup, fixed that.